src/Database/Query.php

```diff
@@ -1869,6 +1870,18 @@ public function newExpr($rawExpression = null): QueryExpression
         return $expression;
     }
 
+    /**
+     * Returns a new instance of `StringExpression`.
+     *
+     * @param string $string String value
+     * @param string|null $collation String collation
+     * @return \Cake\Database\Expression\StringExpression
+     */
+    public function newString(string $string, ?string $collation = null)
```

Comment:
Is this method necessary? Requiring to specify collation doesn't seem like a common enough use case to warrant a wrapper method, that too for a single statement.

Reply:
I was thinking the people who need this would use it a lot. The orm is already very verbose.

Reply:
I think StringExpression should probably make collation required otherwise you wouldn’t need the expression